src/index.html

```diff
@@ -29,6 +29,16 @@
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root"></div>
+    <script>
+      // Dragons 🐉 have been reintroduced into the client, but hear us out:
```

Comment:
Thanks for adding a note!

Let's add a few github issue links to help track this, as well as make a note in the HACKS.md file :).